java-tools/OsmAndMapCreatorUtilities/src/main/java/net/osmand/NativeJavaRendering.java

```diff
@@ -482,60 +485,35 @@ public Map<String, FileIndex> initIndexesCache(File dir,
 		if (filesToUse == null) {
 			filesToUse = new ArrayList<>();
 		}
-			
+		
 		if (dir.exists() && dir.listFiles() != null) {
-			TreeSet<String> allFileNames = new TreeSet<String>();
+			TreeSet<String> allFileNames = new TreeSet<>();
 			for (File obf : Algorithms.getSortedFilesVersions(dir)) {
 				if (!obf.isDirectory() && obf.getName().endsWith(".obf")) {
 					filesToUse.add(obf);
 					allFileNames.add(obf.getName());
 				}
 			}
-			// Collections.reverse(sortedFiles);
-			Iterator<File> fileIt = filesToUse.iterator();
-			while (fileIt.hasNext()) {
-				File obf = fileIt.next();
-				try {
-					FileIndex fileIndex = cache.getFileIndex(obf, true);
-					if (fileIndex != null) {
-						map.put(obf.getAbsolutePath(), fileIndex);
-					}
-					boolean basemapPresent = checkMoreGenericMapPresent(allFileNames, obf.getName());
-					if (filterDuplicates && basemapPresent) {
-						log.debug("SKIP initializing cause bigger map is present: " + obf.getName());
-						fileIt.remove();
-					}
-				} catch (Exception e1) {
-					e1.printStackTrace();
+			
+			List<String> regionNameList = new ArrayList<>();
+			List<File> files = new ArrayList<>();
+			
+			OsmandRegions osmandRegions = new OsmandRegions();
```

Comment:
This code is now added in all variations where jave rendering is initialized in all cases but it's needed likely only for filterDuplicates